test/testbufferoverrun.cpp

```diff
@@ -3863,6 +3864,13 @@ class TestBufferOverrun : public TestFixture {
         ASSERT_EQUALS("", errout_str());
     }
 
+    void pointer_out_of_bounds_5() { // #20337
+        check("int foo(char str[6])\n"
+              "{\n"
```

Comment:
we can save 1 line of code in this test case if you move the `{` up. Please do that.